src/repl.ts

```diff
@@ -135,16 +150,53 @@ export function createRepl(options: CreateReplOptions = {}) {
           _console.error(error);
         }
       } else {
-        err = error;
+        err = error as Error;
```

Comment:
then error declaration should have been `any` instead of `Error | null`, however I can type it `as any` to avoid changing callback signatures.

Reply:
Regarding to the question, yes. It can be a non `Error`:

```typescript
} else {
  console.log(typeof error);
  err = error as any;
}
```

```typescript
> throw 123
number
Uncaught 123
```